File: perl/circle.pl
#!/usr/bin/env perl

use strict;
use warnings;

print("Enter circle radius: ");

my $radius = <STDIN>;

my $c = $radius * 3.1415 * 2;
my $a = $radius * $radius * 3.1415;

print("Circle radius is: $c\n");
print("Circle area is: $a\n");

File: cpp/circle.cpp
#include <iostream>

using namespace std;

int main(void)
{
    float i;
    cout << "Enter circle radius: ";
    cin >> i;
    cout << "Circumfence: " << i * 3.1415f * 2 << endl;
    cout << "Area: " << i * i * 3.1415f << endl;
    return 0;
}
